docs/articles-jp/apex/interactive-debugger.md

```diff
@@ -69,9 +69,9 @@ VS Code で初めて Apex デバッガを使用するときは、次の設定手
 
 デバッグセッション中に例外が発生した場合に、Apex デバッガの実行を停止するには、例外にブレークポイントを設定します。例外ブレークポイントに達すると、例外の原因となったコード行でデバッガが一時停止します。[Debug \(デバッグ\)] ビューの [Call Stack \(コールスタック\)] パネルに、例外の名前が表示されます。
 
-例外ブレークポイントを設定するには、Ctrl+Shift+P キー \(Windows、Linux\) または Cmd+Shift+P キー \(macOS\) を押してコマンドパレットを開き、**[Apex Debug: Configure Exceptions \(Apex デバッグ: 例外を設定\)]** を選択します。使用可能な例外のリストに、[exceptions in the `System` namespace](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_exception_methods.htm) と、`Exception` を拡張するプロジェクトの Apex クラスが表示されます。リストから例外を選択して、**[Always break \(常に中断\)]** をクリックします。
```

Comment:
@ntotten We noticed that we have some Japanese text going on in these files. In updating the label text for this palette feature, we are wondering how to go about making sure the Japanese text gets updated properly as well. At your convenience, please let us know your thoughts on this

Thank you,
Cade